### PR TITLE
Add multiplication feature

### PR DESCRIPTION
**Pull request by sriramsantosh**

### CODE REVIEW

File: multiplication.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+def multiply(x,y):
+    return x*y
```
Comment (aripirala) [inline multiplication.py:2]: Can we handle more cases within multiply function? thi sis too simplistic

**Review by aripirala — CHANGES_REQUESTED** [review]
Need more changes

File: multiplication.py (hunk 1 above)
Comment (sriramsantosh) [reply multiplication.py:2]: For now, this is good.

Comment (aripirala) [reply multiplication.py:2]: Fine.

**Review by aripirala — APPROVED** [review]

**Pull request merged**